Dockerfile.acceptance-testing

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.4
+
+EXPOSE 8080
+
+COPY ./inventory /usr/bin/
+COPY ./config.yaml /usr/bin/
+
```

Comment:
you can add `STOPSIGNAL SIGINT` instead of `docker kill -s SIGINT..`

Also since we're mounting TESTS_DIR into the container at `/tests` you can dump coverprofile to `/tests/coverage-acceptance.txt` and get rid of `docker cp ..`